mapie/classification.py

```diff
@@ -1287,149 +726,26 @@ def predict(
         alpha_np = cast(NDArray, alpha)
         check_alpha_and_n_samples(alpha_np, n)
 
-        y_pred_proba = self.estimator_.predict(X, agg_scores)
-        y_pred_proba = self._check_proba_normalized(y_pred_proba, axis=1)
-        if agg_scores != "crossval":
-            y_pred_proba = np.repeat(
-                y_pred_proba[:, :, np.newaxis], len(alpha_np), axis=2
-            )
-
-        # Choice of the quantile
-        if self.method == "naive":
-            self.quantiles_ = 1 - alpha_np
+        # Estimate prediction sets
+        if self.method == "raps":
```

Comment:
Same remark as above : this could be included directly in the non conformity score if we perform the raps split their

Reply:
Done 👍 I hope the proposal suits you